### PR TITLE
Implement infra nginx(MAPCO-3400)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                     |

Further  information:

### CODE REVIEW

File: helm/config/default.conf [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (CL-SHLOMIKONCHA) [inline helm/config/default.conf:1]: remove first empty line

File: helm/values.yaml [hunk 2]
```diff
@@ -100,24 +62,44 @@ resources:
...
+        name: '{{ .Release.Name }}-nginx-configmap' 
+
+  extraVolumeMounts: 
+    - name: nginx-config
+      mountPath: "/etc/nginx/conf.d/default.conf"
+      subPath: default.conf
+    - mountPath: "/etc/nginx/nginx.conf"
+      name: nginx-config
+      subPath: nginx.conf
+  route:
```
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:93]: add ingress section aswell

File: helm/config/default.conf (hunk 1 above)
Comment (almog8k) [reply helm/config/default.conf]: done

File: helm/config/default.conf [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{{- $chartName := include "export-management.name" . -}}
+{{- $serviceName := include "service.fullname" . -}}
+{{- $releaseName := .Release.Name -}}
+
+map $msec $nanosec {
```
Comment (asafMasa) [inline helm/config/default.conf:5]: check with roee and ronen if needed as it should be in infra nginx

File: helm/config/default.conf [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    location = /opa {
+      internal;
+
+      proxy_set_header Content-Type application/json;
+      proxy_pass {{ .Values.nginx.authorization.url }};
+    }    
+    {{ end }}
+}
```
Comment (asafMasa) [inline helm/config/default.conf:57]: end of file

File: helm/Chart.yaml [hunk 5]
```diff
@@ -4,3 +4,7 @@ description: A Helm chart for export-management service
 type: application
 version: 1.1.4
 appVersion: 1.1.4
+dependencies:
```
Comment (asafMasa) [inline helm/Chart.yaml:7]: add "common-" prefix to the name of the "export-management" and it's nginx

File: helm/values.yaml [hunk 6]
```diff
@@ -100,24 +64,51 @@ resources:
...
+    repository: nginx-otel-unprivileged
+    tag: 'v1.0.0' 
+  port: 8080
+  internalServicePort: 80
+  targetPort: 8080
+  servicePort: 8080
+  nodePort: 30001
+  authorization:
+    enabled: true
+    domain: 'raster'
```
Comment (asafMasa) [inline helm/values.yaml:80]: this should include other domains also (DEM and 3D)

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job 👍

File: helm/config/default.conf [hunk 7]
```diff
@@ -0,0 +1,53 @@
+{{- $chartName := include "export-management.name" . -}}
```
Comment (ronenkapelian) [inline helm/config/default.conf:1]: remove if it not in use

File: helm/config/default.conf [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+{{- $serviceName := include "service.fullname" . -}}
+{{- $releaseName := .Release.Name -}}
```
Comment (ronenkapelian) [inline helm/config/default.conf:3]: remove if it not in use

File: helm/config/default.conf [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    keepalive_timeout  500;
+    proxy_connect_timeout       600;
+    proxy_send_timeout          600;
+    send_timeout                600;
+    client_max_body_size        5000;
+    client_header_timeout       600;
+    client_body_timeout         600;
+    client_header_buffer_size   12288; # 12K
+    large_client_header_buffers 4 12288; # 12K
+    fastcgi_read_timeout        300;
```
Comment (ronenkapelian) [inline helm/config/default.conf:23]: fastgi not relevant I think, Remove and validate correctness

File: helm/config/nginx.conf [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+    worker_connections  1024;
+}
+
+
+http {
+    opentelemetry_config /otel/otel-nginx.toml;
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format json escape=json
```
Comment (ronenkapelian) [inline helm/config/nginx.conf:21]: consult with Roee, on geoserver i used the default log_format of infra nginx, I think you should use it too, on future we will remove it also from pycsw the specific format

**Review by ronenkapelian — CHANGES_REQUESTED** [review]
few notes to resolve

File: helm/config/default.conf (hunk 7 above)
Comment (almog8k) [reply helm/config/default.conf:1]: done

File: helm/config/default.conf (hunk 8 above)
Comment (almog8k) [reply helm/config/default.conf:3]: done

**Review by ronenkapelian — APPROVED** [review]
Be calm! :1st_place_medal:

**Review by asafMasa — APPROVED** [review]

**Pull request merged**